### PR TITLE
Blog post about BYOD and BYOS

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
@sanjaysrikakulam as discussed, your contribution would be appreciated

many thanks!

### CODE REVIEW

**Comment by sanjaysrikakulam** [comment]
I have added a couple of sections showing how to create and integrate a bucket with Galaxy and how to use BYOS to run a workflow. Please let me know if you need anything else.

File: content/news/2024-09-20-esg-byos-im/index.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: "EuroScienceGateway project: Bring Your Own Storage with Galaxy"
```
Comment (bgruening) [inline content/news/2024-09-20-esg-byos-im/index.md:2]:
```suggestion
title: "Bring Your Own Storage with Galaxy"
```

I think this is general enough.

File: content/news/2024-09-20-esg-byos-im/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+<img class="center" src="byos-add-minio-bucket-to-galaxy.png" width="80%" alt="Add your MinIO bucket to Galaxy.">
+
+Once the bucket is added to Galaxy the user can start using it as the default storage location or select it at different levels: per `History`, per `Tool`, and `Workflow`.
+
+<img class="center" src="byos-your-newly-added-external-storage.png" width="80%" alt="Newly added external storage.">
+
+# Running workflows in Galaxy with Bring Your Own Storage
+
+The user can now run tools and workflows in Galaxy using the newly configured external storage. Below is an example of a workflow that performs the assembly of bacterial genomes using [Shovill](https://usegalaxy.eu/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fshovill%2Fshovill%2F1.1.0%20galaxy2) a faster _de novo_ assembly pipeline for Illumina paired end reads.
```
Comment (bgruening) [inline content/news/2024-09-20-esg-byos-im/index.md:112]:
```suggestion
The user can now run tools and workflows in Galaxy using the newly configured external storage. Below is an example of a workflow that performs the assembly of bacterial genomes using [Shovill](https://usegalaxy.eu/?tool_id=shovill) a faster _de novo_ assembly pipeline for Illumina paired end reads.
```

Using short ID?

File: content/news/2024-09-20-esg-byos-im/index.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+    <img class="center" src="successful-completion-of-the-workflow-invocation.png" width="80%" alt="Successful completion of workflow run">
+
+    the user can find all the output files in the external storage.
+
+    <img class="center" src="all-workflow-datasets-now-in-the-bucket.png" width="80%" alt="Output files in external storage.">
+
+6. The user can also evaluate it by clicking the `Dataset details` of a job and observe that the user's external storage is used as the `Dataset Storage`
+
+    <img class="center" src="dataset-storage-location.png" width="80%" alt="Dataset details showing external storage.">
```
Comment (bgruening) [inline content/news/2024-09-20-esg-byos-im/index.md:137]: Can you please ack the original PR at the end?

**Review by bgruening — COMMENTED** [review]
BYOD is missing at the moment here, isn't it? Maybe adding a cross-link to the BYOD - the GTN/onedata post we had?

File: content/news/2024-09-20-esg-byos-im/index.md (hunk 3 above)
Comment (sebastian-luna-valero) [reply content/news/2024-09-20-esg-byos-im/index.md]: Added now.

**Comment by bgruening** [comment]
Please take it out of Draft if you are happy :)

**Comment by sebastian-luna-valero** [comment]
Thanks for your review @bgruening 

Yes, I am happy as it is; however, in order to address your comment about BYOD:
https://galaxyproject.org/news/2024-05-27-gtn-training-data-onedata/
https://training.galaxyproject.org/training-material/news/2024/06/03/onedata.html

I have discussed with @sanjaysrikakulam and he's going to coordinate with you ;)

**Comment by sanjaysrikakulam** [comment]
> BYOD is missing at the moment here, isn't it? Maybe adding a cross-link to the BYOD - the GTN/onedata post we had?

We previously discussed that there will be two separate posts, and this is one of them, which centers around EGI. I will write another one that covers everything related to BYOS and BYOD. Please let me know if this has changed; I can add a section about BYOD and include a MinIO example here.

**Comment by bgruening** [comment]
Separate posts are good. We have also https://github.com/galaxyproject/training-material/issues/5295 ... we should not add so many screenshots to the training as they are outdated quickly, but large parts of those posts can be in the training as well.

Thanks!!!

**Comment by sanjaysrikakulam** [comment]
> Separate posts are good. We have also [galaxyproject/training-material#5295](https://github.com/galaxyproject/training-material/issues/5295) ... we should not add so many screenshots to the training as they are outdated quickly, but large parts of those posts can be in the training as well.
> 
> Thanks!!!

Nice! I was unaware of this issue because I was not watching that repo (I have now enabled it and subscribed to that issue and the repo). Yes, I will write the blog post in the coming days by taking what we already have in this post, ESG deliverable, and adding the missing ones. We can then use it for the training material.

**Pull request merged**